src/main/java/org/folio/service/impl/EcsTlrServiceImpl.java

```diff
@@ -68,6 +69,24 @@ public boolean delete(UUID requestId) {
     return false;
   }
 
+  @Override
+  public void updateRequestItem(UUID secondaryRequestId, UUID itemId) {
+    log.debug("updateRequestItem:: parameters secondaryRequestId: {}, itemId: {}", secondaryRequestId, itemId);
+    ecsTlrRepository.findBySecondaryRequestId(secondaryRequestId).ifPresentOrElse(
+      ecsTlr -> updateItemIfChanged(ecsTlr, itemId),
+      () -> log.error("updateRequestItem: ECS TLR with secondary request ID {} not found", secondaryRequestId));
```

Comment:
I think this might be a totally valid case. Are we sure we want to log it in ERROR level?